### PR TITLE
Dependencies update

### PR DESCRIPTION
**Pull request by fat23cat**
- The latest Angular 8 updates
- The latest auth0 updates
I've got rid of 3 high priority vulnerabilities, but 3 low priority is still alive, it will be fixed as soon as auth0 contributors update their package dependencies.

### CODE REVIEW

**Review by MocanaAtBroad — DISMISSED** [review]

File: ddp-workspace/projects/ddp-sdk/package.json [hunk 1]
```diff
@@ -17,11 +17,14 @@
     "@nguniversal/express-engine": "8.1.1",
     "@ngx-translate/core": "12.1.2",
     "@ngx-translate/http-loader": "4.0.0",
-    "auth0-js": "9.13.2",
-    "auth0-lock": "11.26.0",
+    "auth0-js": "9.14.0",
+    "auth0-lock": "11.26.3",
+    "bootstrap": "3.4.1",
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-sdk/package.json:22]: For A-T?

File: ddp-workspace/projects/ddp-sdk/package.json [hunk 2]
```diff
@@ -17,11 +17,14 @@
...
     "classlist.js": "1.1.20150312",
     "core-js": "2.5.4",
+    "font-awesome": "4.7.0",
     "hammerjs": "2.0.8",
+    "jquery": "3.5.1",
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-sdk/package.json:27]: For bootstrap?

File: ddp-workspace/projects/toolkit/package.json [hunk 3]
```diff
@@ -17,12 +17,15 @@
...
+    "auth0-js": "9.14.0",
+    "auth0-lock": "11.26.3",
+    "bootstrap": "3.4.1",
     "classlist.js": "1.1.20150312",
     "core-js": "2.5.4",
     "ddp-sdk": "1.0.0",
+    "font-awesome": "4.7.0",
     "hammerjs": "2.0.8",
+    "jquery": "3.5.1",
     "kaop-ts": "3.1.2",
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/toolkit/package.json:29]: Same question. We need bootstrap, jquey and font-awesome now?

Comment (fat23cat) [reply ddp-workspace/projects/toolkit/package.json:29]: All these dependencies are needed for AT.

Comment (MocanaAtBroad) [reply ddp-workspace/projects/toolkit/package.json:29]: 👍

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**